declarations_site/chatbot/utils.py

```diff
@@ -19,6 +26,8 @@
 def ukr_plural(value, *args):
     value = int(value)
     rem = value % 10
+    if value > 100:
```

Comment:
if seems to be redundant here:
In [1]: 10 % 100
Out[1]: 10

In [2]: 15 % 100
Out[2]: 15